pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_events.go

```diff
@@ -53,27 +57,22 @@ func TestDuplicatedEventForUpgrade(events monitorapi.Intervals, kubeClientConfig
 func TestDuplicatedEventForStableSystem(events monitorapi.Intervals, clientConfig *rest.Config) []*junitapi.JUnitTestCase {
 
 	evaluator := duplicateEventsEvaluator{
-		allowedRepeatedEventPatterns: AllowedRepeatedEventPatterns,
-		allowedRepeatedEventFns:      AllowedRepeatedEventFns,
-		knownRepeatedEventsBugs:      KnownEventsBugs,
+		allowedDupeEvents: AllowedPathologicalEvents,
 	}
 
-	operatorClient, err := operatorv1client.NewForConfig(clientConfig)
+	etcdAllowance, err := newDuplicatedEventsAllowedWhenEtcdRevisionChange(context.TODO(), clientConfig)
 	if err != nil {
-		panic(err)
-	}
-	etcdAllowance, err := newDuplicatedEventsAllowedWhenEtcdRevisionChange(context.TODO(), operatorClient)
-	if err != nil {
-		panic(fmt.Errorf("unable to construct duplicated events allowance for etcd, err = %v", err))
-	}
-	evaluator.allowedRepeatedEventFns = append(evaluator.allowedRepeatedEventFns, etcdAllowance.allowEtcdGuardReadinessProbeFailure)
```

Comment:
As most of the others, this was not hooked up to anything. I see it actively failing, so this one remains. In the new framework it boils down to just a bump to the threshold. It also looks like it was never added to the upgrade suite, which seemed wrong, it now exists in both.